src/main/java/consoleapp/eventadapters/EventController.java

```diff
@@ -84,6 +85,17 @@ public EventInfo getEventByName(String name) {
     }
 
     public List<HashMap<String, String>> getEvents() {
-        return eventGetter.getEvents();
+        List<HashMap<String, String>> hashMapList = new ArrayList<>();
+        List<EventInfo> eventInfos = eventGetter.getEvents();
+        for (EventInfo info : eventInfos) {
+            HashMap<String, String> infoMap = new HashMap<>();
```

Comment:
The id of event might be needed?

Reply:
The previous implementation had the `EventController` return a hashmap without the ids. I didn't want to change that coz this pr was just about changing the interfaces of request boundaries.

Reply:
Right, I thought it has something to do with gui